ArcGISRuntimeSDKQt_SampleViewer/ArcGISRuntimeSDKQt_Samples.pro

```diff
@@ -139,7 +139,7 @@ QMAKE_TARGET_COPYRIGHT = Copyright 2017 Esri Inc.
 include(samples.pri)
 
 # contains source files for zlib-ng and minizip-ng
-include($$PWD/../../3rdparty/zlib_minizip_ng.pri)
+include($$PWD/../3rdparty/zlib_minizip_ng.pri)
```

Comment:
This alone could bring back the path-too-long problems on Windows. The build is very touchy.

We should still try to do what makes sense.

Reply:
We just moved the project down a directory so all the relative paths need updating. I would of thought reducing a level would help alleviate the problem rather than add too it.

Reply:
I'd have thought the same thing but the build machine was generating really long relative paths, so this could actually add to the `../../../` depending on where it sits in the path. We'll see what happens when we go to merge it. If we hit problems again, we can consider the workaround I mentioned in our internal issue.